Soccar [Game-Server]/src/nl/soccar/gameserver/model/command/listener/BroadcastCommandListener.java

```diff
@@ -0,0 +1,32 @@
+package nl.soccar.gameserver.model.command.listener;
+
+import nl.soccar.gameserver.controller.socnet.message.ChatMessage;
+import nl.soccar.gameserver.model.PlayerWrapper;
+import nl.soccar.gameserver.model.command.Command;
+import nl.soccar.gameserver.model.command.CommandListener;
+import nl.soccar.gameserver.model.session.RoomWrapper;
+import nl.soccar.library.enumeration.Privilege;
+
+/**
+ * @author Lesley
+ */
+public final class BroadcastCommandListener extends CommandListener {
+
+    public BroadcastCommandListener() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)